desktop/apps/auction/components/layout/auction_info/AuctionInfoDesktop.js

```diff
@@ -78,7 +81,12 @@ AuctionInfoDesktop.propTypes = {
 }
 
 const mapStateToProps = (state) => {
-  const { auction } = state.app
+  const { auction, isEcommerceSale } = state.app
+
+  let upcomingLabel = auction.upcomingLabel()
+  if (isEcommerceSale) {
+    upcomingLabel = capitalize(upcomingLabel.replace('Bidding', '').trim())
```

Comment:
I'm ~97% sure this is sufficient.

Reply:
This worries me a little bit-- maybe we can update the actual [method](https://github.com/artsy/force/blob/master/desktop/models/sale.coffee#L180-L191) to check for the `is_auction` property and return the correct copy?